bitrise.yml

```diff
@@ -2,29 +2,29 @@ format_version: 1.3.1
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 app:
   envs:
-  - RELEASE_VERSION: 1.1.0
+    - RELEASE_VERSION: 1.1.0
 workflows:
   audit-this-step:
     steps:
-    - script:
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -ex
-            stepman audit --step-yml ./step.yml
+      - script:
+          inputs:
+            - content: |-
+                #!/bin/bash
+                set -ex
+                stepman audit --step-yml ./step.yml
   deps-update:
```

Comment:
please update this workflow to use go dep, as a rewference please see xcode-archive step for example